chartmogul/api/customer_note.py

```diff
@@ -0,0 +1,29 @@
+from marshmallow import Schema, fields, post_load, EXCLUDE
+from ..resource import Resource
+from collections import namedtuple
+
+
+class CustomerNote(Resource):
+    """
+    https://dev.chartmogul.com/v1.0/reference#customer-notes
+    """
+
+    _path = "/customer_notes{/uuid}"
+    _root_key = "entries"
+    _many = namedtuple("CustomerNotes", [_root_key, "has_more", "cursor"])
+
+    class _Schema(Schema):
+        uuid = fields.String()
+        customer_uuid = fields.String(allow_none=True)
+        type = fields.String(allow_none=True)
```

Comment:
I'm not sure if we should set `allow_none=True` for these fields. Customer note should always belong to a customer and the `type` should always be there